.circleci/config.yml

```diff
@@ -114,6 +114,9 @@ jobs:
       - run:
           name: Test amplify
           command: xcodebuild test -enableThreadSanitizer NO -workspace Amplify.xcworkspace -scheme Amplify -sdk iphonesimulator -destination "${destination}" | xcpretty --simple --color --report junit
+      - run:
+          name: Upload coverage report to Codecov
+          command: bash ~/amplify-ios/.circleci/codecov.sh # From https://codecov.io/bash 
```

Comment:
Should I add `-F nameOfTask` to show the coverage data in Github without clicking into `codecov.io` website to view full report like this [one]( https://github.com/aws-amplify/amplify-ios/pull/720/commits/53ee1a55bad83bdc61f436ea9da1debc9beaaff0#diff-1d37e48f9ceff6d8030570cd36286a61R152)?

Reply:
Yes. Also consider adding the `-J <package name>` option to speed up coverage reports